python/lsst/jointcal/associations.cc

```diff
@@ -36,46 +37,47 @@ namespace lsst {
+        cls.def("fittedStarListSize", &Associations::fittedStarListSize);
+        cls.def("associateCatalogs", &Associations::associateCatalogs, "matchCutInArcsec"_a = 0,
+                "useFittedList"_a = false, "enlargeFittedList"_a = true);
+        cls.def("collectRefStars", &Associations::collectRefStars, "refCat"_a, "matchCut"_a, "fluxField"_a,
+                "refCoordinateErr"_a, "rejectBadFluxes"_a = false);
+        cls.def("deprojectFittedStars", &Associations::deprojectFittedStars);
+        cls.def("nCcdImagesValidForFit", &Associations::nCcdImagesValidForFit);
+        cls.def("nFittedStarsWithAssociatedRefStar", &Associations::nFittedStarsWithAssociatedRefStar);
 
-    cls.def("createCcdImage", &Associations::createCcdImage);
-    cls.def("addCcdImage", &Associations::addCcdImage);
-    cls.def("prepareFittedStars", &Associations::prepareFittedStars);
-    cls.def("cleanFittedStars", &Associations::cleanFittedStars);
+        cls.def("createCcdImage", &Associations::createCcdImage);
+        cls.def("addCcdImage", &Associations::addCcdImage);
+        cls.def("prepareFittedStars", &Associations::prepareFittedStars);
+        cls.def("cleanFittedStars", &Associations::cleanFittedStars);
 
-    cls.def("getCcdImageList", &Associations::getCcdImageList, py::return_value_policy::reference_internal);
-    cls.def_property_readonly("ccdImageList", &Associations::getCcdImageList,
-                              py::return_value_policy::reference_internal);
+        cls.def("getCcdImageList", &Associations::getCcdImageList, py::return_value_policy::reference_internal);
+        cls.def_property_readonly("ccdImageList", &Associations::getCcdImageList,
+                                  py::return_value_policy::reference_internal);
 
-    cls.def("computeBoundingCircle", &Associations::computeBoundingCircle);
+        cls.def("computeBoundingCircle", &Associations::computeBoundingCircle);
 
-    cls.def("getCommonTangentPoint", &Associations::getCommonTangentPoint);
-    cls.def("setCommonTangentPoint", &Associations::setCommonTangentPoint);
-    cls.def("computeCommonTangentPoint", &Associations::computeCommonTangentPoint);
+        cls.def("getCommonTangentPoint", &Associations::getCommonTangentPoint);
+        cls.def("setCommonTangentPoint", &Associations::setCommonTangentPoint);
+        cls.def("computeCommonTangentPoint", &Associations::computeCommonTangentPoint);
 
-    cls.def("getEpoch", &Associations::getEpoch);
-    cls.def("setEpoch", &Associations::setEpoch);
+        cls.def("getEpoch", &Associations::getEpoch);
+        cls.def("setEpoch", &Associations::setEpoch);
+    });
 }
+}  // namespace
 
-PYBIND11_MODULE(associations, mod) {
-    py::module::import("lsst.jointcal.ccdImage");
-    py::module::import("lsst.sphgeom");
-    declareAssociations(mod);
+void wrapAssociations(lsst::cpputils::python::WrapperCollection &wrappers) {
+    wrappers.addInheritanceDependency("lsst.sphgeom");
```

Comment:
This should be in the main `_jointcalLib.cc`.

Reply:
Moved